UQ-eLIPSE/crucible
# feature/ELIPSE-888

Test

1. Checkout the branch.
2. Go to the server.
3. Run yarn lint.

No issues/warnings should be reported.
